mains/parse_metadata.py

```diff
@@ -98,3 +100,34 @@
     X_tmp, Y_tmp, train_size=size_val, random_state=1
 )
 
+#load images
+def load_image(addr):
+    img = cv.imread(addr)
+    #img = cv.resize(img, (224, 224), interpolation=cv.INTER_CUBIC)
+    img = cv.cvtColor(img, cv.COLOR_BGR2RGB)
+    img = img.astype(np.float32)
+    return img
+
+
+#convert data to features
+def _int64_feature(value):
+    return tf.train.Feature(int64_list=tf.train.Int64List(value=[value]))
+
+def _bytes_feature(value):
+    return tf.train.Feature(bytes_list=tf.train.BytesList(value=[value]))
+
+
+#write data in tfrecords file
+train_filename = 'train.tfrecords'
```

Comment:
(maybe you should just execute this piece of code explicitly.
)[https://stackoverflow.com/questions/419163/what-does-if-name-main-do]
```python
if __name__ == "__main__":
```

Reply:
The script is used only once to train the initial network, I doubt we'll ever call it from anywhere else except maybe a setup bash script or something like that.